src/main/java/seedu/taassist/logic/commands/DeletecCommand.java

```diff
@@ -4,11 +4,15 @@
 import static seedu.taassist.commons.core.Messages.MESSAGE_MODULE_CLASS_DOES_NOT_EXIST;
 import static seedu.taassist.logic.parser.CliSyntax.PREFIX_MODULE_CLASS;
 
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 
 import seedu.taassist.logic.commands.exceptions.CommandException;
 import seedu.taassist.model.Model;
 import seedu.taassist.model.moduleclass.ModuleClass;
+import seedu.taassist.model.student.Student;
 
```

Comment:
I think the constructor should also perform a null check with `requireAllNonNull()`
 (Github doesnt allow reviewing a line that wasn't changed so this comment is here)

Reply:
I find that there are other command constructors having the same problem. Fixed in the next commit.